src/utils.js

```diff
@@ -0,0 +1,4 @@
+export const getWikipediaAttrFromUrl = url => {
+	const matches = /^https?:\/\/(\w+)(\.m)?\.wikipedia\.org\/wiki\/(\w+)/.exec( url )
```

Comment:
If an editor goes to the site and copies the URL from the browser, they may end up with something like `https://en.wikipedia.org/w/index.php?title=Cat`, that should also be supported.